apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx

```diff
@@ -78,10 +83,28 @@ export function BrowseCompany() {
       ...latestQuery,
       skills: [],
       desiredPositions: [],
+      employmentTypes: [],
+      federalStates: [],
       isJobFair2022Participant: undefined,
     }))
   }
 
+  const shouldShowFilters = useMemo(
```

Comment:
Might this be a case of [premature optimization](https://kentcdodds.com/blog/usememo-and-usecallback)?

Reply:
Yeah I guess you're right. Since I'm not passing shouldShowFilters to any component at all, no need to memoize this.